### PR TITLE
Stickers

### LINKED ISSUE #1056: Sticker support
Adding support for [stickers](https://discord.com/developers/docs/resources/sticker).

It looks like the work has been done by @vysiondev in https://github.com/bwmarrin/discordgo/pull/982.

It was mentioned that the PR should be broken up into multiple smaller PRs. I would like to submit a PR with the sticker portion of this update.

### PR DESCRIPTION
**Pull request by Jordan-Cartwright**
Fixes #1056 

Adds sticker support

### CODE REVIEW

**Comment by Jordan-Cartwright** [comment]
Is there anything else that I should do with this PR?

cc: @FedorLap2006 @CarsonHoffman

**Comment by FedorLap2006** [comment]
Lgtm, but Imma go through the docs again just to be sure

File: message.go [hunk 1]
```diff
@@ -188,6 +191,50 @@ type File struct {
...
+type StickerType int
+
+// Defines Sticker types.
+const (
+	StickerTypeStandard StickerType = 1
+	StickerTypeGuild    StickerType = 2
+)
+
+// Sticker represents a sticker object that can be sent in a Message.
+type Sticker struct {
```
Comment (QPixel) [inline message.go:214]: It seems like this is missing the type field that is listed in the docs.

**Review by QPixel — CHANGES_REQUESTED** [review]
Besides missing the type field, everything else seems fine to me

File: message.go (hunk 1 above)
Comment (Jordan-Cartwright) [reply message.go:215]: Thank you for making this note, I have updated `message.go` with this change

**Review by QPixel — APPROVED** [review]

**Pull request merged**